File: Ruby/scores.rb
class Scores
  # attr_accessor reads and writes
  # attr_writer writes only
  attr_reader :player1
  attr_reader :player2
  attr_reader :score1
  attr_reader :score2
  def initialize(player1, score1, player2, score2)
    @player1 = player1
    @score1 = score1
    @player2 = player2
    @score2 = score2
  end
end

File: Csharp/ConsoleApp/Scores.cs
namespace ConsoleApp
{
    internal class Scores
    {
        public string Player1 { get; set; }
        public int Score1 { get; set; }
        public string Player2 { get; set; }
        public int Score2 { get; set; }
        public Scores(string player1, int score1, string player2, int score2)
        {
            Player1 = player1;
            Score1 = score1;
            Player2 = player2;
            Score2 = score2;
        }
    }
}
